cmd/mo-service/config.go

```diff
@@ -108,23 +113,64 @@ func (c *Config) createFileService(name string) (fileservice.FileService, error)
 	return nil, fmt.Errorf("file service named %s not set", name)
 }
 
+func (c *Config) createETLFileService(name string) (fileservice.FileService, error) {
```

Comment:
This method is not needed.
Just add a disk-etl FileService entry to the config file